.github/scripts/build.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -e
+
+fpga_fam=$1
+shift
+
+# select particular example for xc7
+examples="$@"
```

Comment:
Why do we need this bit?

Reply:
We use it to "test the CI" locally, see: 
https://github.com/SymbiFlow/symbiflow-examples/pull/88/files#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168fR45